iset.mm

```diff
@@ -99583,6 +99612,769 @@ seq K ( .+ , G , S ) ) $=
+    iseqf1olemfvp.g $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( G ` x ) e. S ) $.
+    iseqf1olemfvp.p $e |- P = ( x e. ( ZZ>= ` M )
+      |-> if ( x <_ N , ( G ` ( f ` x ) ) , ( G ` M ) ) ) $.
+    $( Lemma for ~ iseqf1o .  (Contributed by Jim Kingdon, 30-Aug-2022.) $)
+    iseqf1olemfvp $p |- ( ph
+        -> ( [_ T / f ]_ P ` A ) = ( G ` ( T ` A ) ) ) $=
+      ( cfv wcel wceq syl csb cle wbr cif cv cuz cmpt csbeq2i cvv cfz co wf cfn
+      wf1o f1of cz elfzel1 elfzel2 fzfigd fex syl2anc nfcvd fveq1 fveq2d ifeq1d
+      mpteq2dv csbiegf syl5eq simpr breq1d ifbieq1d elfzuz elfzle2 fveq2 eleq1d
+      wa iftrued ralrimiva ffvelrnd rspcdva eqeltrd fvmptd eqtrd ) ACGFDUAZQCKU
+      BUCZCFQZHQZJHQZUDZWGABCBUEZKUBUCZWJFQZHQZWHUDZWIJUFQZWDEAWDGFBWOWKWJGUEZQ
+      ZHQZWHUDZUGZUAZBWOWNUGZGFDWTPUHAFUIRZXAXBSAJKUJUKZXDFULZXDUMRXCAXDXDFUNXE
+      MXDXDFUOTZAJKAIXDRZJUPRLIJKUQTAXGKUPRLIJKURTUSXDXDUMFUTVAGFWTXBUIXCGXBVBW
+      PFSZBWOWSWNXHWKWRWMWHXHWQWLHWJWPFVCVDVEVFVGTVHAWJCSZVPZWKWEWMWGWHXJWJCKUB
+      AXIVIZVJXJWLWFHXJWJCFXKVDVDVKACXDRZCWORNCJKVLTAWIWGEAWEWGWHAXLWENCJKVMTVQ
+      ZAWJHQZERZWGERBWOWFWJWFSXNWGEWJWFHVNVOAXOBWOOVRAWFXDRWFWORAXDXDCFXFNVSWFJ
+      KVLTVTWAWBXMWC $.
+  $}
+
+  ${
+    iseqf1o.1 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x .+ y ) e. S ) $.
+    iseqf1o.2 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) ->
+                     ( x .+ y ) = ( y .+ x ) ) $.
+    iseqf1o.3 $e |- ( ( ph /\ ( x e. S /\ y e. S /\ z e. S ) ) ->
+                     ( ( x .+ y ) .+ z ) = ( x .+ ( y .+ z ) ) ) $.
+    iseqf1o.4 $e |- ( ph -> N e. ( ZZ>= ` M ) ) $.
+    iseqf1o.6 $e |- ( ph -> F : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+    iseqf1o.7 $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( G ` x ) e. S ) $.
+
+    ${
+      iseqf1olemstep.k $e |- ( ph -> K e. ( M ... N ) ) $.
+      iseqf1olemstep.j $e |- ( ph
+        -> J : ( M ... N ) -1-1-onto-> ( M ... N ) ) $.
+      iseqf1olemstep.const $e |- ( ph
+        -> A. x e. ( M ..^ K ) ( J ` x ) = x ) $.
+
+      ${
+        $d J u $.  $d K u x $.  $d M u x $.  $d N u $.  $d J x $.  $d Q x $.
+        $d ph x $.
+        iseqf1olemnk $e |- ( ph -> K =/= ( `' J ` K ) ) $.
+        iseqf1olemqres.q $e |- Q = ( u e. ( M ... N ) |->
+          if ( u e. ( K ... ( `' J ` K ) ) ,
+            if ( u = K , K , ( J ` ( u - 1 ) ) ) ,
+            ( J ` u ) ) ) $.
+        iseqf1olemqsumk.p $e |- P = ( x e. ( ZZ>= ` M )
+          |-> if ( x <_ N , ( G ` ( f ` x ) ) , ( G ` M ) ) ) $.
+
+        ${
+          $d .+ x y z $.  $d G f x $.  $d J f v x $.  $d J u v x $.
+          $d J f x y z $.  $d K u v x $.  $d K x y z $.  $d M f x $.
+          $d M u x $.  $d N f x $.  $d N u x $.  $d P v x $.  $d P x y z $.
+          $d Q f v x $.  $d Q f x y z $.  $d S x y z $.  $d ph u v x $.
+          $d ph x y z $.
+          $( Lemma for ~ iseqf1o . ` Q ` gives the same sum as ` J ` in the
+             range ` ( K ... ( ` ' J ` K ) ) ` .  (Contributed by Jim Kingdon,
```

Comment:
Not sure if there should be a space between backtick and single quote.

Reply:
The backtick inside backticks should be quoted. That is the original problem here which probably is what led to that extraneous space (maybe from a rewrap).

Reply:
Now fixed. As the metamath book says "Two consecutive grave accents ‘‘ are treated as a single actual grave
accent (both inside and outside of math mode) and will not cause the output
processor to enter or exit math mode." so the grave accents need to be doubled both in the converse and the function application.

Reply:
git has a few ways to do this, including `git commit --amend` and `git rebase -i`.

Github shows this with an "Outdated" marker next to the comments which refer to a previous version of the pull request.

As for what the preferred practice is, everyone has a somewhat different opinion on this but most of the choices have some downside.

Reply:
Ok, thanks. I often use `git commit --amend -am "message"` before submitting in order not to have too many commits in a given PR. I didn't know one could do it after `git push origin mybranch` and doing a PR.
I still do not master `git rebase`...

Reply:
Oh another piece I forgot to mention is `git push -f` (or `git push --force-with-lease` which is pretty much the same). That is perhaps why you couldn't amend and then change a pull request.

Force push (and rebase) is a bit of a sharp edged tool so I won't object if you choose not to use them.